src/Microsoft.IdentityModel.Logging.Abstractions/LogEntry.cs

```diff
@@ -0,0 +1,52 @@
+﻿//------------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.IdentityModel.Logging.Abstractions
+{
+    /// <summary>
+    /// Defines the structure of a log entry.
+    /// </summary>
+    public class LogEntry
+    {
+        /// <summary>
+        /// Defines the <see cref="EventLevel"/>.
+        /// </summary>
+        public EventLevel EventLevel { get; set; }
+
+        /// <summary>
+        /// Message to be logged.
+        /// </summary>
+        public string Message { get; set; }
+
+        /// <summary>
+        /// A unique identifier for a request that can help with diagnostics across components.
+        /// </summary>
+        public string CorrelationId { get; set; }
```

Comment:
>CorrelationId [](http://example.com/codeflow?start=22&length=13)

Consider renaming to "ActivityId' or including a remark that notes this is referred to as "ActivityId" in other parts of the code (e.g. CallContext)

Reply:
I'll add a remark. Let me know if renaming the property to ActivityId would be more appropriate for consistency. This decision was made to remain consistent up the stack.

Reply:
Added a remark [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1823/files#:~:text=Also%20referred%20to%20as%20ActivityId%20in%20Microsoft.IdentityModel.Tokens.CallContext.)